OpenRA.Game/Traits/World/Shroud.cs

```diff
@@ -79,6 +82,14 @@ public Shroud(Actor self)
 			explored = new CellLayer<bool>(map);
 		}
 
+		void INotifyCreated.Created(Actor self)
+		{
+			fogEnabled = self.World.LobbyInfo.GlobalSettings.Fog;
+			var shroudEnabled = self.World.LobbyInfo.GlobalSettings.Shroud;
+			if (!shroudEnabled)
```

Comment:
A reason why you pulled this out into it's own `var` instead of keeping it like in the first commit?
(`if (!self.World.LobbyInfo.GlobalSettings.Shroud)`.)